packages/amplication-build-manager/src/build-runner/build-runner.controller.ts

```diff
@@ -5,22 +5,23 @@ import { Controller, Post } from "@nestjs/common";
 import { ConfigService } from "@nestjs/config";
 import { EventPattern, Payload } from "@nestjs/microservices";
 import axios from "axios";
-import { plainToInstance } from "class-transformer";
 import { Env } from "../env";
 import { BuildRunnerService } from "./build-runner.service";
 import { CodeGenerationFailureDto } from "./dto/CodeGenerationFailure";
-import { CodeGenerationRequestDto } from "./dto/CodeGenerationRequest";
 import { CodeGenerationSuccessDto } from "./dto/CodeGenerationSuccess";
 import {
   CodeGenerationFailure,
+  CodeGenerationRequest,
   CodeGenerationSuccess,
 } from "@amplication/schema-registry";
+import { DsgCatalogService } from "./dsg-catalog.service";
 
 @Controller("build-runner")
 export class BuildRunnerController {
   constructor(
-    private readonly buildRunnerService: BuildRunnerService,
     private readonly configService: ConfigService<Env, true>,
+    private readonly buildRunnerService: BuildRunnerService,
+    private readonly dsgCatalogService: DsgCatalogService,
```

Comment:
I think you need to add these providers to the tests

Reply:
Done!